server/routes/charts.js

```diff
@@ -158,12 +159,41 @@ router.route('/charts/:chart_id')
               return currentSite;
             }, {}))
         )
+
+      const legendquery = [
+        {
+          $match : { _id : new ObjectID(chart_id) }
+        },
+        {
+          $project : {
+            _id : 0.0,
+            fieldLabelValueMap : 1.0,
+          }
+        },
+        {
+          $unwind: { path: '$fieldLabelValueMap' }
+        }
+      ]
+      const getFieldValues = await dataDb
```

Comment:
nitpick: I'd rename this to something like `fieldValues` since it's returning the array. I'd expect `getFieldValues` to be a method.